tox.ini

```
@@ -38,3 +38,47 @@ commands = pre-commit run --all-files --show-diff-on-failure
 deps = twine
 commands_pre =
 commands = twine check {distdir}/*
+
+[testenv:build-docs]
+basepython = python3
+deps =
+  -r{toxinidir}/docs/requirements.txt
+  # FIXME: re-enable the "-r" + "-c" paradigm once the pip bug is fixed.
+  # Ref: https://github.com/pypa/pip/issues/9243
+  # -r{toxinidir}/docs/requirements.in
+  # -c{toxinidir}/docs/requirements.txt
+description = Build The Docs
+commands =
+  # Retrieve possibly missing commits:
+  -git fetch --unshallow
+  -git fetch --tags
+
+  # Build the html docs with Sphinx:
+  {envpython} -m sphinx \
+    -j auto \
+    -b html \
+    {tty:--color} \
+    -a \
+    -n -W --keep-going\
+    -d "{temp_dir}/.doctrees" \
+    . \
+    "{envdir}/docs_out"
+
+  # Print out the output docs dir and a way to serve html:
+  -{envpython} -c\
+  'import pathlib;\
+  docs_dir = pathlib.Path(r"{envdir}") / "docs_out";\
+  index_file = docs_dir / "index.html";\
+  print("\n" + "=" * 120 +\
+  f"\n\nDocumentation available under:\n\n\
+  \tfile://\{index_file\}\n\nTo serve docs, use\n\n\
+  \t$ python3 -m http.server --directory \
+  \N\{QUOTATION MARK\}\{docs_dir\}\N\{QUOTATION MARK\} 0\n\n" +\
+  "=" * 120)'
+changedir = {toxinidir}/docs
+isolated_build = true
+passenv =
+    SSH_AUTH_SOCK
```

Dedent this to have 2-space indent like in the settings above
```suggestion
  SSH_AUTH_SOCK
```